Configuration/Shared/Workspace-Deployment-Targets.xcconfig

```diff
@@ -1,6 +1,6 @@
 // Base Deployment Targets
 IPHONEOS_DEPLOYMENT_TARGET = 9.0
-MACOSX_DEPLOYMENT_TARGET = 10.10
+MACOSX_DEPLOYMENT_TARGET = 10.15
```

Comment:
Remind me, why did we need to go to 15 and not 14?

Reply:
I don't actually remember, it may have been something that I've since removed. I changed it to 10.14 and everything still seems to compile and run. We'd need to run it on 10.14 to make sure the included version of JavaScriptCore supports all the language features we target though.

Reply:
We'll keep an eye out for this as needed, we may need to add some `#if` gating, but I don't want to limit what version of macOS people using it for apps can support because of codegen requirements.